recipes/libbsd/all/conanfile.py

```diff
@@ -34,13 +34,16 @@ def build_requirements(self):
 
     def export_sources(self):
         export_conandata_patches(self)
-    
+
     def configure(self):
         if self.options.shared:
            self.options.rm_safe("fPIC")
         self.settings.rm_safe("compiler.libcxx")
         self.settings.rm_safe("compiler.cppstd")
-    
+
+    def requirements(self):
+        self.requires("libmd/1.1.0")
```

Comment:
Could you please make that conditional and only add it for newer versions than 0.10.0?

Reply:
Done!